packages/block-library/src/tag-cloud/edit.js

```diff
@@ -82,6 +89,20 @@ function TagCloudEdit( { attributes, setAttributes, taxonomies } ) {
 					max={ MAX_TAGS }
 					required
 				/>
+				<NumberControl
```

Comment:
I think it will be better to have these controls in one row. You can wrap them with `<Flex>` component and then each `NumberControl` should be wrapped with `<FlexItem isBlock>` component.

Reply:
Wrapping fields inside Flex looks more clean and consistent UI. Thanks for this. :smile: